desktop/cmp/grid/Grid.js

```diff
@@ -227,12 +224,14 @@ export class Grid extends Component {
             selModel.select(rec);
         }
         if (!rec) selModel.clear();
-        const {count} = selModel;
+
+        const count = selModel.count,
+            selectedRecs = selModel.records;
 
         // Prepare each item
         const items = menu.items;
         items.forEach(it => {
```

Comment:
Thinks this change makes sense -- seems more flexible to not tie the action to a selectionModel specifically.
I still wonder whether we would want to make prepareFn and actionFn take a config object to make them more flexible for gaining additional info in the future (e.g. column, cell, gridModel).  I am ok with waiting on that change if we feel we want to move forward with this now.

Reply:
I have the change for passing a config object to prepareFn and actionFn on another branch, along with other changes to refine the RestGrid "actions" api as we had discussed. Agree that those changes don't need to hold up the action col being merged.

Reply:
Perfect! -- if we can just unwind the intermediary RestGrid API changes in this branch, I would be fine to merge